README.md

```diff
@@ -140,7 +140,7 @@ If you believe you have found a security vulnerability in this project or any of
 
 If you would like to contribute to this project, review [these guidelines](./CONTRIBUTING.md).
 
-To all contributors, we thank you!  Without your contribution, this project would not be what it is today.  We also host a community project page dedicated to [Project Name](<LINK TO https://opensource.newrelic.com/projects/... PAGE>).
+To all contributors, we thank you!  Without your contribution, this project would not be what it is today.  We also host a community project page dedicated to [New Relic Next.js instrumentation](https://opensource.newrelic.com/projects/newrelic/newrelic-node-nextjs).
```

Comment:
I could not find this in the open source website, so unsure if that has merged yet. So can't confirm this link is correct. If it has been merged, then need to find the real link but I'd expect this to be the link.

Reply:
it appears this hasn't been released yet. looking at their PRs they don't ship every PR but cut releases so it'll go in next one